js/src/views/Signer/store.js

```diff
@@ -88,20 +88,18 @@ export default class SignerStore {
   }
 
   fetchLocalTransactions = () => {
```

Comment:
The name suggests that the transactions will be fetched, but actually it sets up a subscription.
The name should be altered and it should be ensured that the method is never called more than once.